R/bootstrap.nlsfit.R

```diff
@@ -1120,10 +1120,41 @@ residual_plot.bootstrapfit <- function (x, ..., error_fn = sd, operation = `/`)
   prediction_boot <- do.call(rbind, lapply(1:nrow(x$t), prediction_boot_fn))
   
   residual_val <- operation(x$y, prediction_val)
+  # We want to subtract or divide (depending on the given `operation`) the
```

Comment:
Very funny... One line would have been enough...

Reply:
I've made it a one-liner now.

Reply:
Shame, I liked the outburst. Probably better this way though...

Reply:
Oh, I read your comment as ironic. I've reverted the reverting, so we have it in the code now.